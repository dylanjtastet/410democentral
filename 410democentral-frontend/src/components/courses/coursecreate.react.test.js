import React from 'react';
import renderer from 'react-test-renderer';
import Coursecreate from './coursecreate.jsx';


test('Component responds to text input changes', ()=>{
    const component = renderer.create(
        <Coursecreate></Coursecreate>
    );

    let tree = component.toJSON()
    expect(tree).toMatchSnapshot()

    tree.children[0].props.onChange({target:{value:"hello"}})
        
    expect(tree).toMatchSnapshot()
})