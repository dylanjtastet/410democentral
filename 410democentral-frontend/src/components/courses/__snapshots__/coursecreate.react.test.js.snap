// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component responds to text input changes 1`] = `
<div>
  <input
    className="input paraminputs textinput"
    onChange={[Function]}
    placeholder="course name"
    type="text"
    value=""
  />
  <div>
    <span
      className="rightmargin"
    >
      <button
        className="button is-static"
      >
        Add
      </button>
    </span>
    <span>
      <button
        className="button"
        onClick={[Function]}
      >
        Cancel
      </button>
    </span>
  </div>
</div>
`;

exports[`Component responds to text input changes 2`] = `
<div>
  <input
    className="input paraminputs textinput"
    onChange={[Function]}
    placeholder="course name"
    type="text"
    value=""
  />
  <div>
    <span
      className="rightmargin"
    >
      <button
        className="button is-static"
      >
        Add
      </button>
    </span>
    <span>
      <button
        className="button"
        onClick={[Function]}
      >
        Cancel
      </button>
    </span>
  </div>
</div>
`;
